apps/sandbox/locust/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.coverage.run]
+branch = true
+source = [ "sandbox_locust" ]
+
+[tool.coverage.report]
+exclude_lines = [ "if TYPE_CHECKING:" ]
+show_missing = true
+
+[tool.poetry]
+name = "sandbox-locust"
+version = "1.0.0"
+description = ""
+authors = [ ]
+license = "Proprietary"
+readme = "README.md"
+
+  [[tool.poetry.packages]]
+  include = "sandbox_locust"
+
+  [tool.poetry.dependencies]
+  python = "3.10.14"
+  locust = "2.31.8"
+
+[tool.poetry.group.dev.dependencies]
+autopep8 = "2.0.2"
+flake8 = "6.0.0"
+pytest = "7.3.1"
+pytest-sugar = "0.9.7"
+pytest-cov = "4.1.0"
+pytest-html = "3.2.0"
+
+[tool.locust]
+# locustfile = ""
+headless = true
```

Comment:
consider having a setting in the future to allow UI to be used without risking accidental check-in of changes to this variable.

Reply:
`--headful` and `--headless` can be used to override

Reply:
The task `sandbox-locust:serve` now runs Locust headful. `--headless` is now specified in the CI workflow.